__tests__/config.test.js

```diff
@@ -2,15 +2,16 @@ const { getImageConfig, getScreenshotConfig } = require('../src/config');
 
 describe('config', () => {
   it('getImageConfig', () => {
-    const config = {
+    const config = { taskOptions: {
       imageConfig: {
         threshold: 0.5,
-        bar: 'should be ignored',
+        bar: 'should be included',
       },
       foo: 'should be ignored',
-    };
+    }};
 
     expect(getImageConfig(config)).toEqual({
+      bar: 'should be included',
       createDiffImage: true,
       resizeDevicePixelRatio: true,
       threshold: 0.5,
```

Comment:
Strange that the original test expectation is exactly the opposite of what we want...

This all looks fine. I'm going to test cadence with it then should be good to tag.

Reply:
This test seems to check that only the "expected" keys were extracted from the combined command/plugin config, whereas what the plugin expects and what `cy.screenshot` supports aren't the same. A test to test that the written lines work, not that the functionality works.